sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -498,7 +498,7 @@ abstract class AbstractSync(
         nodesToSave: MutableList<Node>,
         savedNodeHandlers: MutableList<suspend (node: Node) -> Unit>
     ) {
-        logger.info("Syncing incoming for issue ${issue.rawId} $timelineItem ${timelineItem.identification()}")
+        //logger.info("Syncing incoming for issue ${issue.rawId} $timelineItem ${timelineItem.identification()}")
```

Comment:
why not reduce the logging level (e.g. to TRACE) instead of commenting it out completely?
or if you think it is completely unnecessary, remove it